pkg/antctl/command_definition.go

```diff
@@ -320,6 +326,64 @@ func respTransformer(obj interface{}) (interface{}, error) {
 	return target, nil
 }
 
+// format the table output for "get" commands
+func (cd *commandDefinition) tableOutputForGetCommands(obj interface{}, writer io.Writer) error {
+	var list []common.TableOutput
+	if reflect.TypeOf(obj).Kind() == reflect.Slice {
+		s := reflect.ValueOf(obj)
+		if s.Len() == 0 || s.Index(0).Interface().(common.TableOutput) == nil {
+			var buffer bytes.Buffer
+			buffer.WriteString("\n")
+			if _, err := io.Copy(writer, &buffer); err != nil {
+				return fmt.Errorf("error when copy output into writer: %w", err)
+			}
+			return nil
+		}
+		for i := 0; i < s.Len(); i++ {
+			ele := s.Index(i)
+			list = append(list, ele.Interface().(common.TableOutput))
+		}
+	} else {
+		ele, _ := obj.(common.TableOutput)
+		list = []common.TableOutput{ele}
+	}
+
+	// get the elements and headers of table
+	args := list[0].GetTableHeader()
+	rows := make([][]string, len(list)+1)
+	widths := make([]int, len(args))
+	rows[0] = args
+	for i, element := range list {
+		rows[i+1] = element.GetTableRow(maxTableOutputColumnLength)
+	}
+
+	// get the width of every column
+	for j := 0; j < len(args); j++ {
+		width := len(rows[0][j]) + 1
+		for i := 1; i < len(list)+1; i++ {
+			if width < len(rows[i][j])+1 {
+				width = len(rows[i][j]) + 1
+			}
+		}
+		widths[j] = width
```

Comment:
nit: you can first not + 1 in L362-L365 and  + 1 here only.

Reply:
done